tools/replay.cpp

```diff
@@ -482,20 +479,58 @@ class Replay {
     }
 };
 
-int main(int ac, char** av)
+static cxxopts::ParseResult parse(int argc, char* argv[])
 {
-  if ( ac < 2 || ac > 3 )
-    Replay::usage_and_exit( "Incorrect number of program arguments" );
+  try
+  {
+    cxxopts::Options options(argv[0], "Replay an umpire session from a file");
+
+    options
+      .add_options()
+      (  "h, help", "Print help")
+      (  "i, infile"
```

Comment:
Is the second argument the long-form option?

Reply:
Yep.  Here is what the help output looks like:

``` script
$ ./bin/tools/replay -h
Replay an umpire session from a file
Usage:
  ./bin/tools/replay [OPTION...]

  -h, --help           Print help
  -i, --infile FILE    Input file created by Umpire library with
                       UMPIRE_REPLAY=On
  -t, --testfile FILE  Test file created used for testing replay only
```

Reply:
Can you fix this message to make more sense: `Test file created used for testing replay only`, or see if you can "hide" that option?

Reply:
@davidbeckingsale, The -t option now has english text and has been hidden since it is used for testing purposes.